tree-writer/src/renderers/sell_a_bee/mod.rs

```diff
@@ -0,0 +1,36 @@
+use std::cmp::min;
+
+use image::io::Reader as ImageReader;
+use image::imageops::{self};
+use image::{Rgb};
+
+use std::env;
+
+use super::{Pixel, TreeCanvas};
+
+// Name: Sell a bee
+// Description: Celebi's got some new moves in this render!
+// Author: Liam Henderson
+
+pub fn draw(tick: u64) -> TreeCanvas {
+    let mut canvas = TreeCanvas::new();
+
+    let frame_num = tick % 249 + 1;
+    let frame_name = format!("tree-writer/src/renderers/sell_a_bee/right resolution/{:0>4}.png", frame_num);
+    let frame = ImageReader::open(frame_name).unwrap().decode().unwrap().resize_exact(20, 75, imageops::FilterType::Gaussian);
```

Comment:
So basically this will fail since the files at `tree-writer/src/renderers/sell_a_bee/right resolution/{:0>4}.png` wouldn't be accessible on the tree, they're loaded at compile time.

I think the best way to fix this would be to write a script that can take image sets like this and create a json file with the data, and use `include_str!()` to "bundle" it with the binary.